src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -68,9 +68,11 @@ def validate_location(cmd, location):
             break
 
 
-def get_chart_path(registry_path, kube_config, kube_context):
+def get_chart_path(registry_path, kube_config, kube_context, container_registry_username="", container_registry_password=""):
```

Comment:
I think it would be better to keep the default param values of **_container_registry_username_** and **_container_registry_password_** to None (None would imply unset strings here.) , and call the **_helm_login_** function only when both is not None.

Reply:
ack